frontend/web/components/Toolip.js

```diff
@@ -16,7 +16,7 @@ const Tooltip = class extends React.Component {
           <span className='ion ion-ios-help' data-for={this.id} data-tip />
         )}
         <ReactTooltip
-          html
+          html={false}
```

Comment:
We only want to disable html for some usages, a lot of the times we use tooltips e.g. permissions uses html for tooltip content.

Reply:
Implementing DOMPurify to sanitize the tooltip content since it removes XSS scripts but not HTML content.

Reply:
As per our conversation, proceeded to HTML encode the tag labels to escape HTML and display content as plain text.
Also, converted Tooltip code to Typescript functional component.